### ISSUE TITLE
Markdown export (Pod V2) does not add .md suffix to file names in links

### ISSUE THREAD

**Issue by fnurl** [dendronhq/dendron#2647]
## Please select the area your bug applies to. (Multiple selections are Possible. You can leave blank if you're not sure.)

- [ ] Workspace.  VSCode workspace, vaults, Intellisense/autocomplete, Dendron settings
- [ ] Lookup.  Dendron's Lookup Command
- [ ] Views.  Dendron Preview, Tree View, Side Panels in the UI
- [ ] Schema.  Dendron Schemas
- [x] Pod.  Data import from / export to Dendron
- [ ] Publish.  External Site Publish
- [ ] Markdown.  Markdown features, such as syntax support and features

## Describe the bug

When wikilinks are converted into markdown links, the filename extension `.md` is omitted.

## To Reproduce

1. Export a note containing a wikilink using `Dendron: Export V2`

Here is my config for reference:

```
podId: export-vault 
podType: MarkdownExportV2
wikiLinkToURL: false
destination: /path/to/my/export/folder
convertTagNotesToLinks: false
convertUserNotesToLinks: false
addFrontmatterTitle: false
```

## Expected behavior

`[[note1]]` --> `![note1 title](/note1.md)` what I get is `[[note1]]` --> `![note1 title](/note1)`

## Desktop (please complete the following information)

- OS: macOS 12.3
- Browser (if with published websites; include version): [e.g. chrome, safari]
- Versions (all that apply to bug):
  - Dendron Extension: 0.87
  - VS Code version: 1.65.2
  - Dendron CLI:

## Additional context

N/A

## Dendron Log file

N/A

**Comment by SeriousBug** [comment]
Looking at the code, this seems to be the intended behavior for the markdown export pod.

@Harshita-mindfire Do you know why the markdown export pod omits the `.md` portions from the links?

**Comment by Harshita-mindfire** [comment]
This will be fixed when https://github.com/dendronhq/dendron/pull/2771 releases next week!

**Comment by Harshita-mindfire** [comment]
This should be fixed with Dendron v0.92. Feel free to comment here if that's not the case.
